src/NuGet.Clients/NuGet.VisualStudio.Common/ProjectSystems/IProjectSystemCache.cs

```diff
@@ -109,7 +110,7 @@ public interface IProjectSystemCache
         /// <param name="projectNames">Primary key.</param>
         /// <param name="projectRestoreInfo">The project restore info including tools.</param>
         /// <returns>True if operation succeeded.</returns>
-        bool AddProjectRestoreInfo(ProjectNames projectNames, DependencyGraphSpec projectRestoreInfo);
+        bool AddProjectRestoreInfo(ProjectNames projectNames, DependencyGraphSpec projectRestoreInfo, IReadOnlyList<IAssetsLogMessage> additionalMessages);
```

Comment:
This is a public interface, what is the policy of changing the interfaces?

Reply:
The project is `NuGet.VisualStudio.Common`. Like the other `src\NuGet.Client\*` projects, we don't publish this as a package, meaning no-one can have an API or ABI dependence on this project. API breaking changes are only important for `src\NuGet.Core\*` projects, and our `IVs*` interfaces.